api/tools/src/main/java/org/pmiops/workbench/tools/ConfigLoader.java

```diff
@@ -28,14 +27,12 @@ public class ConfigLoader extends Tool {
 
   private static final Logger log = Logger.getLogger(ConfigLoader.class.getName());
 
-  private static final ImmutableMap<String, Class<?>> CONFIG_CLASS_MAP =
-      ImmutableMap.of(
+  private static final Map<String, Class<?>> CONFIG_CLASS_MAP =
```

Comment:
It's immutable in the new version too, just not obvious from the signature, thanks to `Map.of()`.

The purpose of this change is to move away from a library dependency on Google Guava to its native replacement, available since Java 9.